src/test/regress/expected/pg13.out

```diff
@@ -117,8 +116,6 @@ SELECT normalize(name) FROM text_table ORDER BY 1;
 (2 rows)
 
 INSERT INTO text_table VALUES (normalize(U&'\0061\0308bc', NFC));
-=======
->>>>>>> 96b3a00b8... Changelog: Test unicode escapes
```

Comment:
can this be rebase edited in the previous commit? They are added there and removed here. Would be nice to get it out of the history.